### ISSUE TITLE
Demonstrate active learning makes training more efficient

### ISSUE THREAD

**Issue by RustyPanda** [RustyPanda/zoobot#32]
Goal: comparison plot of active learning vs random selection

I know the model learns well when trained from scratch with access to all images.
I need to show that it can learn more efficiently with active learning selection of images

The simplest way to demonstrate this is to train two models:
- with random or AL selection
- from scratch at each iteration, because we avoid any questions over gradient descent speed/local minima/etc (review later)

The first part is to get a baseline model training poorly, and then as well as the default model (86% after 3k steps of 30x128 images)

This is a much longer train cycle than I've been running on my tests. I may have been stopping too early.

To ensure training does not show fewer images due to insufficient data, I have set training input config to repeat the dataset indefinitely, as needed.

To improve selection, I have modified dropout on prediction (not training or eval) to p=0.25. Intuitively, the predictions are currently a little all over the places as shown by histograms.

Training has somewhat converged after 10,000 steps. 

p2.xlarge trains 130 at steps/min (128->64px images). 8000 steps would require 1 hour ($0.27) of training time.

Training strategy:
- [ ] Shards of 1024 images, with 1 as a training shard and 6 as "unlabelled"
- [ ] 6 iterations of 8000 steps (6h, $2)
- [ ] To go through all images, select 1024 images per cycle
- [ ] To select 1024 images, need to predict on, say, 4 shards. Aim to get confusing images early.

**Comment by RustyPanda** [comment]
- [ ] Clean directory on first run - RunConfig.warm_start=False
- [ ] Do not reload estimator each iteration, but track progress

Do not load estimator from same directory, but also do not delete directory. Perhaps move the estimator directory each iteration?
